modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/healthcheck/LocalHealthcheck.scala

```diff
@@ -13,20 +13,25 @@ import cats.{Applicative, Show}
 import scala.concurrent.duration._
 import scala.util.control.NoStackTrace
 
+import io.constellationnetwork.node.shared.domain.cluster.services.Session
 import io.constellationnetwork.node.shared.domain.cluster.storage.ClusterStorage
 import io.constellationnetwork.node.shared.domain.healthcheck.LocalHealthcheck
 import io.constellationnetwork.node.shared.http.p2p.clients.NodeClient
+import io.constellationnetwork.node.shared.http.p2p.middlewares.TimeoutMiddleware.withTimeout
 import io.constellationnetwork.schema.cluster.SessionToken
 import io.constellationnetwork.schema.peer._
 
 import io.chrisdavenport.mapref.MapRef
+import org.http4s.client.Client
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 import retry._
 
 object LocalHealthcheck {
   def make[F[_]: Async: Supervisor](nodeClient: NodeClient[F], clusterStorage: ClusterStorage[F]): F[LocalHealthcheck[F]] = {
     def mkPeersR = MapRef.ofConcurrentHashMap[F, PeerId, F[Fiber[F, Throwable, Unit]]]()
-    def retryPolicy: RetryPolicy[F] = RetryPolicies.fibonacciBackoff[F](2.seconds)
+    def retryPolicy: RetryPolicy[F] = RetryPolicies.limitRetries[F](3).join(RetryPolicies.constantDelay[F](5.seconds))
+    // TODO: Ensure new retry policy to prevent dead forks does not cause issue, previously using:
+    // def retryPolicy: RetryPolicy[F] = RetryPolicies.fibonacciBackoff[F](2.seconds)
```

Comment:
This may be unnecessary, I think test still works without this, but these 2-3 day fibonnaci retries are just wrong to begin with.